modules/op-chains-generating-cosign-secret.adoc

```diff
@@ -0,0 +1,70 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="chains-generating-cosign-secret_{context}"]
+= Generating the cosign key pair by using the TektonConfig CR
+
+To use the `cosign` signing scheme for {tekton-chains} secrets, you can generate a `cosign` key pair that uses ECDSA encryption by setting the `generateSigningSecret` field in the `TektonConfig` custom resource (CR) to `true`.
```

Comment:
Will the intended audience know what ECDSA stands for? If not, consider spelling it out.

Reply:
It will, especially with the word "encryption".